src/controllers/hid/hidcontroller.cpp

```diff
@@ -151,10 +151,8 @@ void HidController::guessDeviceCategory() {
     if (hid_interface_number==-1) {
         if (hid_usage_page==0x1) {
             switch (hid_usage) {
-                case 0x2: info = tr("Generic HID Mouse"); break;
```

Comment:
This change seem to be unrelated. 
Now it runs to the default branch which is less describtive.

Reply:
These devices do not appear in the GUI because they are filtered out in HidEnumerator.